structure/extension/_RDOC.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.2
+---
+contact: https://github.com/glamberson/gedcom-evidence
+extension tags: _RDOC
+label: Research Documentation
+lang: en-US
+payload: XREF
```

Comment:
The value of the payload for a pointer should be the URI it points to, like `@<https://github.com/glamberson/gedcom-evidence/_RACT>@` (but replacing that with the correct URI, I'm not sure what it's pointing to.